docs/index.asciidoc

```diff
@@ -16,58 +12,31 @@ data from sources like logs and metrics or IT operations
 analytics and security analytics.
 
 ECS is still under development and backward compatibility is not guaranteed. Any
-feedback on the general structure, missing fields, or existing fields is appreciated.
-For contributions please read the  https://github.com/elastic/ecs/blob/master/CONTRIBUTING.md[Contribution Guidelines].
+feedback on the general structure, missing fields, or existing fields is
+appreciated. For contributions please read the
+https://github.com/elastic/ecs/blob/master/CONTRIBUTING.md[Contribution
+Guidelines].
 
 [float]
-[[versions]]
-=== Versions
-
-The master branch of this repository should never be considered an
-official release of ECS. You can browse https://github.com/elastic/ecs/releases[official releases] of ECS.
-
-Please note that when the README.md file and other generated files
-(like schema.csv and template.json) are not in agreement,
-the README.md should be considered the official spec.
-The other two files are simply provided as a convenience, and may not always be
-fully up to date.
-
-/////
-Working TOC
-
-ECS Intro
-  Versions
-  MINI TOC???
-Fields (generated)
-  Base
-  Agent
-  Cloud
-  ETC
-* Use Case header = Use Cases (generated)
-* Implementing
-* About (FAQs)
-* Contributing
-* GLOSSARY? 
+[[ecs-field-types]]
+=== Types of fields
 
-* Mike's stuff (https://docs.google.com/document/d/1srylXQDgO0z5rbwho1o8nKIxr9icTTl_nXZGYbElxYs/edit#heading=h.y1xeds3o3jae)
+* *Core fields.* Fields that are most common across all use cases. 
+Focus on populating these fields first. 
 
-ECS Definitions and Entity Relationships
-** ECS data model
-** Definitions
-** Top-Level Namespaces/Objects
-** Reusable Namespaces/Objects
-** Assets and Asset Lists
-** Pseudonymized and Anonymized Data in ECS
-** Threat and Vulnerability Data in ECS
-** Revisit inbound, outbound bytes/packets
+* *Extended fields.* Any fields that are not a core field. 
+Extended fields may apply to more narrow use cases, or may be more open
+to interpretation depending on the use case. Extended fields are more likely to
+change over time.
```

Comment:
Love the very direct and straightforward mention of what core and extended mean, right in the ECS index page ❤️

Reply:
@MarkSettleES let me know that  "core" and "custom"  have been proposed instead of "core" and "extended". 
WDYT @webmat? @MikePaquette? @MarkSettleES? Others?

Reply:
Core and extended are absolutely both the main levels we use to define ECS fields.

However Mark makes a good point. I have seen some of Mike's content list the 3 together, and I think that's a great idea. I think it could reinforce the idea that ECS is a standard that still embraces the custom that people need around it.

I agree we should add it as a third bullet point in there.

Reply:
@karenzone @MarkSettleES to close the loop here,  "Custom" can be included as a third level, but is not an alternative to "extended".

So I agree with @webmat let's go with three levels:
1. Core
2. Extended
3. Custom